pkg/util/servicediscovery/registry.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
+
+package servicediscovery;
+
+import "github.com/gogo/protobuf/gogoproto/gogo.proto";
+
+option (gogoproto.marshaler_all) = true;
+option (gogoproto.unmarshaler_all) = true;
+
+message ServiceInstanceDesc {
+	string id = 1;
+	int64 timestamp = 2;
```

Comment:
What exactly is the timestamp? In the ring it's UnixSeconds, here it's UnixNanos. Should be documented, to avoid confusion.